mergeKSortedLists.py

```diff
@@ -0,0 +1,25 @@
+"""
+Time complexity: O(nlogn) We insert n nodes in a heap (logn per insertion)
+Space complexity: O(n) where n is the number of lists
```

Comment:
Please take different variables for number of lists and no. of elements in list to be more specific. Space complexity will not be equal to the number of lists